Vagrantfile

```diff
@@ -82,6 +82,10 @@ Vagrant.configure("2") do |config|
     v.memory = 2048
     v.cpus = 2
     v.linked_clone = true
+    # Enable Nested Hardware Virtualisation - requires VirtualBox 6
+    v.customize ["modifyvm", :id, "--nested-hw-virt", "on"]
+    # Use the recommended paravirtualization interface for windows (hyperv) - requires VirtualBox 6
+    v.customize ["modifyvm", :id, "--paravirtprovider", "hyperv"]
```

Comment:
Do you know if that breaks something using VirtualBox on Mac or Linux?

Reply:
I tried it only on Mac - there it worked perfectly